Anilist | 1.1.0

TheNetsky opened the pull request:
- Fixed manual updates not saving
- Fixed user avatar not displaying
- Some formatting
- Fixed default status not saving
- Bumped to 1.1.0

DanielKovalevich commented on .github/workflows/main.yml:
```diff
@@ -0,0 +1,38 @@
+on:
+  push:
+    tags:
+    - 'v*'
```
We want the action to execute whenever a pull request has been accepted and merged

ConradWeiser replied:
In order for releases to be published along with artifacts, it is a __requirement__ that you have a tag associated. After a lot of experimentation, I've discovered that there's no way to get around this, unfortunately. 

One of my experiments was to create a tag called 'release', and to simply hardcode each build release, as something under that tag. It worked once. It didn't work a second time, not allowing an overwrite. 

The bottom line, there is no one step solution to make everything run appropriately on pull request. You can get close by manually assigning tags through a git client of some sort BEFORE passing the request, although at that point, why use the merge trigger in general. You'd have more fine control over the release process if you stick with just tag triggers.

ConradWeiser replied:
Tagging system has been replaced with on push or pull request resolution

DanielKovalevich commented on .github/workflows/main.yml:
```diff
@@ -0,0 +1,38 @@
+          node-version: ${{ matrix.node-version }}
+
+      - run: npm install
+      - run: npm test
```
The tests will take too long in our case. Normally, this would not be an issue but we will need to temporarily disable this functionality.

ConradWeiser commented:
Branch is temporarily dirty, do not merge

ConradWeiser commented:
The release generation Workflow has been swapped out with [a Git Commit action](https://github.com/stefanzweifel/git-auto-commit-action), this will build, and publish the latest artifacts in your repositories `bundles/` directory.

This runs on push, and on git merge. 

The links should be static to access.

TheNetsky committed:
Anilist | 1.1.0

- Fixed manual updates not saving
- Fixed user avatar not displaying
- Some formatting
- Fixed default status not saving
- Bumped to 1.1.0
